src/Controller/ContentController.php

```diff
@@ -49,10 +57,13 @@ public function defaultAction()
      *
      * @return Response
      */
+    #[Route('/testportal')]
```

Comment:
these must be removed https://github.com/pimcore/demo/blob/bc8ba0a6114b4cf9f095f7d54569117f33346cff/src/Controller/ContentController.php#L55-L56